Marlin/Marlin_main.cpp

```diff
@@ -499,27 +497,28 @@ static bool drain_queued_commands_P() {
  */
 void enqueuecommands_P(const char* pgcode) {
   queued_commands_P = pgcode;
```

Comment:
Do we still want to set this when `serial_count == 0`, even though `drain_queued_commands_P()` won't be called?